backend/src/schema/resolvers/posts.js

```diff
@@ -110,14 +110,15 @@ export default {
 
       return post
     },
-    UpdatePost: async (object, params, context, resolveInfo) => {
+    UpdatePost: async (_parent, params, context, _resolveInfo) => {
       const { categoryIds } = params
       delete params.categoryIds
       params = await fileUpload(params, { file: 'imageUpload', url: 'image' })
       const session = context.driver.session()
 
       let updatePostCypher = `MATCH (post:Post {id: $params.id})
-      SET post = $params
+      SET post += $params
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-09-16T10:27:22Z" title="Monday, September 16th 2019, 12:27:22 pm +02:00">Sep 16, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -110,14 +110,15 @@ export default {
 
       return post
     },
-    UpdatePost: async (object, params, context, resolveInfo) => {
+    UpdatePost: async (_parent, params, context, _resolveInfo) => {
       const { categoryIds } = params
       delete params.categoryIds
       params = await fileUpload(params, { file: 'imageUpload', url: 'image' })
       const session = context.driver.session()
 
       let updatePostCypher = `MATCH (post:Post {id: $params.id})
-      SET post = $params
+      SET post += $params
```


Yeah !!!
Wonderful you have found out @mattwr18 . 😘

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-09-16T12:31:47Z" title="Monday, September 16th 2019, 2:31:47 pm +02:00">Sep 16, 2019</time>_

---


haha thanks @Tirokk